### ISSUE TITLE
[Suggestion] Separate icons for "Available from Tess Everis" for Silver and Bright Dust purchases

### ISSUE THREAD

**Issue by DiscountWizard** [joshhunt/destinySets#221]
It would be helpful to be able to see at a glance how many items are available for purchase through Bright Dust each week.

**Comment by Jakosaur** [comment]
Yeah, I was thinking about this when looking through the new items to make the sets. Would be really useful with the new Eververse system 👍🙂

Maybe have the symbol of [Bright Dust](https://www.bungie.net/common/destiny2_content/icons/00b8fd588be5678ad3f94d75dc6a3b19.png) & [Silver](https://www.bungie.net/common/destiny2_content/icons/f7ddbbe7fb59b0abc19029e7e75770c2.png)

@joshhunt the API shows the cost of each Eververse item right?

**Comment by Jakosaur** [comment]
I can really see why you wanted this now, there's a lot on sale each week!

Maybe have the symbol of [Bright Dust](https://www.bungie.net/common/destiny2_content/icons/00b8fd588be5678ad3f94d75dc6a3b19.png) & [Silver](https://www.bungie.net/common/destiny2_content/icons/f7ddbbe7fb59b0abc19029e7e75770c2.png) and just the normal Purchasable Symbol in the odd chance it's available with both Silver and Bright Dust.

I would look at trying to implement this, but I can't get a local build of destinySets working. I am limited to GitHub's online editor so can't really test any changes 😅

**Comment by joshhunt** [comment]
this is a great idea - it would be (reasonably) trivial for now to just not show the $ icon for items that a purchasable for Silver. But then also off the top of my head it shouldn't be too hard to show which currency its purchasable with.

If I get a moment I might take a look at this.

separate note @Jakosaur - what problem are you having getting the code up and running locally? Hit me up on twitter or discord (josh#4793) if you want to go over it!

**Comment by Jakosaur** [comment]
Yeah just ignoring items available for Silver will be perfect. Just need to make sure somehow that if the item is available for both Silver & Bright Dust, that it still shows as purchasable, if the item hasn't been acquired before. Probably a low chance, or never possible to happen but it might 😂

Just added you on Discord, just the same issue as before with "yarn start", but "yarn install" shows something isn't compatible. Hopefully we can get this sorted out, grown a hatred for the GitHub editor 😂

![image](https://user-images.githubusercontent.com/20144356/60766607-f3171380-a0a3-11e9-8e85-adf0c2f6ded4.png)
![image](https://user-images.githubusercontent.com/20144356/60766616-0f1ab500-a0a4-11e9-8028-cf0652e5458f.png)

**Comment by Jakosaur** [comment]
If someone is to look into showing a different icon for Silver purchases & Bright Dust purchases, this week would be a good time to implement it and test as Micro Mini is available for Bright Dust & Silver. (According to DIM).

> Probably a low chance, or never possible to happen but it might 😂

If that's the chosen way to display it, **if** you can display the icons given by Bungie, it would look like this -

$ - Available from [Vendor] (other vendor's for-sale items)
$ - Available from Tess Everis for Bright Dust & Silver
[Bright Dust](https://www.bungie.net/common/destiny2_content/icons/00b8fd588be5678ad3f94d75dc6a3b19.png) - Available from Tess Everis for Bright Dust
 [Silver](https://www.bungie.net/common/destiny2_content/icons/f7ddbbe7fb59b0abc19029e7e75770c2.png) - Available from Tess Everis for Silver 

As you said Josh, this could be a way forward but thought I'd bring up the Micro Mini situation this week 🙂

>  it would be (reasonably) trivial for now to just not show the $ icon for items that a purchasable for Silver.

**Comment by Jakosaur** [comment]
The more I've thought about this, @joshhunt what you said would be easier with what's currently in-place. 

- Don't show $ for an item available for Silver
- Bright Dust purchases will still show the $
- Double check that if an item is available for Bright Dust and Silver that it still shows $

------------------------------------------------------

Not neccessary, but I think it'd be cool to show the currency requirement for all items (if there's a currency attached), not just Eververse. 

E.G: "Available from [Vendor] for [Currency]". 

Maybe items available for Silver just don't show the $ but still have "Available from Tess Everis for Silver". If an item is available for two currencies, "for [Currency1] & [Currency2]".

------------------------------------------------------

I've tried having a go to get something changed regarding this, but I think this is out my league. I'll admit, I got a little lost 😅
